linux/system/repo.sls

```diff
@@ -1,8 +1,15 @@
 {%- from "linux/map.jinja" import system with context %}
 {%- if system.enabled %}
 
-include:
-- linux.system.package
+linux_repo_prereq_pkgs:
```

Comment:
i would just move the linux_packages here ... look at map jinja it contain all necessary requirements